### ISSUE TITLE
button-card error after install

### ISSUE THREAD

**Issue by Paulie-Moo** [dwainscheeren/dwains-lovelace-dashboard#255]
Hi There,
Tried to install your dash, edited yaml files as instructed and I am getting two of these red cards and not much else.
Can you give me any pointers?

> Custom element doesn't exist: button-card.
> type: 'custom:button-card'
> template: more_page_list-item
> name: Continue and load dashboard
> icon: 'mdi:chevron-right'
> styles:
>   card:
>     - padding: 8px
> tap_action:
>   action: call-service
>   service: dwains_dashboard.installed

and

> Custom element doesn't exist: button-card.
> type: 'custom:button-card'
> template: more_page_list-item
> name: Go to documentation for configuration
> icon: 'mdi:file-document'
> styles:
>   card:
>     - padding: 8px
> tap_action:
>   action: url
>   url_path: 'https://dwainscheeren.github.io/dwains-lovelace-dashboard/

Thank you
Paul

**Comment by dwainscheeren** [comment]
Try to clear your browser cache

**Comment by Paulie-Moo** [comment]
Doh!  And that was one of this firt things the instructions said, Im a dickhead - sorry

**Comment by AalianKhan** [comment]
can you please close the issue as it has been resolved

**Comment by Paulie-Moo** [comment]
Oops

**Comment by whoamicrypto** [comment]
Hi, I am getting the same error message ("button-card not installed"). I have clear my cache. Any suggestion what to do? Thanks
Ruediger

**Comment by dwainscheeren** [comment]
DD 3.* doesnt use button card anymore so if you use a custom card or addon please install button card in hacs

**Comment by whoamicrypto** [comment]
Hi. Thx for fast reply. Maybe it's me being unexperienced .. but I did not succeed in fixing the problem. Here a bit more detailed description
1. I am using DD3.3.0
2. I would like to add a a new blueprint page (specifically the battery-page-color)
3. after adding this via the dialogue to the yaml file, I get two errors (a) button-card not installed and auto-entities not installed.  
4. these errors prohibit to use the blueprint
5.Following your recommendation, I deleted the blueprints.
6. I went to HACS ... but here I do not know exactly what to do. I went to Frontend and installed an arbitrary button-card hoping to fix the problem ... w/o success.
ANY THOUGHTS?
Ruediger
